src/h5web/toolbar/Toolbar.module.css

```diff
@@ -91,37 +92,32 @@
 }
 
 .btn:hover > .btnLike {
-  background-color: var(--secondary-lighter);
-  box-shadow: var(--btn-shadow-idle) var(--btn-shadow-color);
-}
-
-.btn:active > .btnLike {
-  box-shadow: var(--btn-shadow-pressed) var(--btn-shadow-color);
+  background-color: var(--h5w-btn-hover--bgColor, whitesmoke);
+  box-shadow: -1px -1px 2px inset var(--h5w-btn-hover--shadowColor, var(--h5w-btnRaised--shadowColor, gray));
```

Comment:
To maintain the design's coherence, the offsets and blur radii of all the box shadows are now hardcoded (and duplicated if needed).